service-loadbalancer/gce/README.md

```diff
@@ -0,0 +1,18 @@
+# GCE Load Balancers
+
+
+## Disclaimer:
+- This is a **work in progress**.
+- For fault tolerant load balancing of ingress traffic, you need:
+  1. Multiple hosts running load balancers
```

Comment:
Well, checking it in is blocked on https://github.com/kubernetes/contrib/pull/142, but otherwise